crane/container.go

```diff
@@ -122,7 +122,9 @@ func (c *container) Dependencies() *Dependencies {
 	dependencies := &Dependencies{}
 	for _, link := range c.RunParams.Link() {
 		linkParts = strings.Split(link, ":")
-		dependencies.All = append(dependencies.All, linkParts[0])
+		if !dependencies.includesAsKind(linkParts[0], "link") {
```

Comment:
I don't think supporting volumes-from will make sense. Is there any possible use-case for that in docker?
But it would be better to have more user friendly error message then "index out of bounds".